test/fixtures/simple_example/main.tf

```diff
@@ -25,8 +25,36 @@ resource "random_pet" "main" {
 }
 
 module "example" {
-  source = "../../../examples/simple_example"
+  source = "../../../examples/simple"
 
-  project_id  = var.project_id
-  bucket_name = random_pet.main.id
+  project = var.project_id
+  name    = random_pet.main.id
+  network = module.vpc.network_name
+  region  = var.region
+}
+
+module "vpc" {
+  source  = "terraform-google-modules/network/google"
+  version = "~> 2.5"
+
+  project_id   = var.project_id
+  network_name = random_pet.main.id
+  routing_mode = "GLOBAL"
+
+  subnets = [
```

Comment:
The router being created doesn't seem to use any subnet or secondary range, so I would just remove these fields.

Reply:
done